Purchases/Public/Purchases.swift

```diff
@@ -481,6 +481,16 @@ extension Purchases {
         subscriberAttributesManager.setOnesignalID(onesignalID, appUserID: appUserID)
     }
 
+    /**
+     * Subscriber attribute associated with the Airship Channel ID for the user
+     * Required for the RevenueCat Airship integration
+     *
+     * - Parameter onesignalID: nil will delete the subscriber attribute
+     */
+    @objc public func setAirshipChannelID(_ airshipChannelID: String?) {
```

Comment:
sounds like we decided to not use the maybe prefix in public-facing parameters? if we haven't documented that can we add that to the style guide?

Reply:
yeah, let's do it. honestly if it was a more mainstream convention, I'd be all for it, but I worry that it might just confuse folks, and I haven't seen any other APIs who do it that way